src/styles/index.scss

```diff
@@ -182,6 +182,14 @@ body {
     background-color: var(--main-color);
   }
 
+  &:hover {
+    transform: scale(1.1);
+    transition-duration: 1s;
```

Comment:
just a small tip: **always** put the transition properties in the element, not in the :hover pseudoclass because if you do this the css will only work when the hover is active and the "out" part will be left without animation